deploy/longhorn-images.txt

```diff
@@ -1,8 +1,8 @@
-longhornio/csi-attacher:v2.2.1-lh2
-longhornio/csi-node-driver-registrar:v1.2.0-lh1
-longhornio/csi-provisioner:v1.6.0-lh2
-longhornio/csi-resizer:v0.5.1-lh2
-longhornio/csi-snapshotter:v2.1.1-lh2
+k8s.gcr.io/sig-storage/csi-attacher:v3.2.1
+k8s.gcr.io/sig-storage/csi-provisioner:v2.1.2
+k8s.gcr.io/sig-storage/csi-resizer:v1.2.0
+k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.3
+k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.3.0
```

Comment:
Why we don't use the latest releases of attacher (3.3.0), provisioner (v2.2.2), resizer (1.3.0) compatible with 1.18?

Reply:
Using the upstream images is good, but this also possibly brings us another challenge when dealing with vulnerability issues the future. However, these images seem all based on `distroless` image, so that should be fine.

cc @meldafrawi

Reply:
> Why we don't use the latest releases of attacher (3.3.0), provisioner (v2.2.2), resizer (1.3.0) compatible with 1.18?

So we import Kubernetes v1.18 (since many of our users are still using it). This leads to the decision of choosing CSI Spec v1.2 since Kubernetes v1.18 uses CSI Spec v1.2. As a result, all CSI components need to support min CSI Version v1.2

We cannot use provisioner (v2.2.2), resizer (1.3.0). But yeah, we can use attacher (3.3.0). I was looking at [the documentation](https://kubernetes-csi.github.io/docs/external-attacher.html) so I didn't see the latest attached v3.3.o

Reply:
Just take another look at [attacher v3.3.0](https://github.com/kubernetes-csi/external-attacher/releases). Actually, it is too new and has recommended Kubernetes version of 1.22. Should we still want to use it?

Reply:
Thanks, but it's minor for a patch update for now. All good.

Reply:
Let's just keep the version u verified already.